tests/Application/src/Kernel.php

```diff
@@ -15,23 +15,11 @@
 
 use PSS\SymfonyMockerContainer\DependencyInjection\MockerContainer;
```

Comment:
Isn't it possible to have a symlink on this file too?

Reply:
No it's not because of the `$this->getProjectDir()` that return a dirname it makes the tests fail